acceptance/framework/cli/cli.go

```diff
@@ -0,0 +1,31 @@
+package cli
```

Comment:
New part of the acceptance test framework for compiling the CLI once and using it multiple times. Before, acceptance tests were compiling a new instance of the CLI for each time the binary was called.